int32 id
float64 value

# enum: ABS, REL
int16 coord_system

float64 timeout
